guides/v3.4.0/components/the-component-lifecycle.md

```diff
@@ -200,7 +201,7 @@ There are a few things to note about the `didInsertElement()` hook:
   particularly when order of execution is important.
 
 [did-insert-element]: https://www.emberjs.com/api/ember/release/classes/Component/events/didInsertElement?anchor=didInsertElement
-[dollar]: https://www.emberjs.com/api/ember/release/classes/Component/methods/$?anchor=%24
+[element]: https://emberjs.com/api/ember/3.4/classes/Component/properties/element?anchor=element
```

Comment:
Can you change this link to use `/release/` instead of `/3.4/` in the url? Thanks!